ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,78 +16,247 @@
+
+
     EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
+  }
+
+  @Test
+  public void shouldRunCtStatement() throws IOException {
+
+    final List<Pair<String, Statement>> statementPairs = new ArrayList<>();
+
+    final CreateTable ct = new CreateTable(EasyMock.niceMock(QualifiedName.class), Collections.emptyList(), false, Collections.emptyMap());
+    statementPairs.add(new Pair<>("CT", ct));
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(statementPairs);
+    EasyMock.expect(engine.getMetaStore()).andReturn(metaStore);
+    final Map<String, Object> props = new HashMap<>();
+    EasyMock.expect(engine.buildMultipleQueries("CT", ksqlConfig, props))
+        .andReturn(Collections.emptyList());
+
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
+  }
+
+  @Test
+  public void shouldRunSetStatements() throws IOException {
+
+    final List<Pair<String, Statement>> statementPairs = new ArrayList<>();
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(statementPairs);
+    EasyMock.expect(engine.getMetaStore()).andReturn(metaStore);
+
+    final SetProperty setProperty = new SetProperty(Optional.empty(), "name", "value");
+    statementPairs.add(new Pair<>("SET", setProperty));
+
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
+
+    assertThat(standaloneExecutor.getConfigProperties().get("name"), equalTo("value"));
+  }
+
+
+  @Test
+  public void shouldRunUnSetStatements() throws IOException {
+
+    final List<Pair<String, Statement>> statementPairs = new ArrayList<>();
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(statementPairs);
+    EasyMock.expect(engine.getMetaStore()).andReturn(metaStore);
+
+    final SetProperty setProperty = new SetProperty(Optional.empty(), "name", "value");
+    statementPairs.add(new Pair<>("SET", setProperty));
 
-    executor.start();
+    final UnsetProperty unsetProperty = new UnsetProperty(Optional.empty(), "name");
+    statementPairs.add(new Pair<>("UNSET", unsetProperty));
 
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
     EasyMock.verify(engine);
+
+    assertTrue(standaloneExecutor.getConfigProperties().isEmpty());
   }
 
   @Test
-  public void shouldExecutePersistentQueries() {
-    final PersistentQueryMetadata query = EasyMock.niceMock(PersistentQueryMetadata.class);
-    EasyMock.expect(engine.createQueries(anyString(), eq(ksqlConfig)))
-        .andReturn(Collections.singletonList(query));
-    query.start();
-    EasyMock.expectLastCall();
-    EasyMock.replay(query, engine);
+  @SuppressWarnings("unchecked")
+  public void shouldRunCsasStatements() throws IOException {
+
+    final List<Pair<String, Statement>> statementPairs = new ArrayList<>();
+
+    final Query csas1Query = EasyMock.niceMock(Query.class);
```

Comment:
Would be good to go though each of these tests and look for the common code. Move common variables to fields. Move common code to functions, etc.

Reply:
The updated version is much more concise!